Sources/NIO/PendingWritesManager.swift

```diff
@@ -451,12 +456,13 @@ internal enum WriteMechanism {
     case nothingToBeWritten
 }
 
-internal protocol PendingWritesManager {
+internal protocol PendingWritesManager: AnyObject {
```

Comment:
I had to add this to work around `triggerWriteOperations` being an `extension` (since it mutates `self`), making it a `mutating func` didn't do the trick either.